sdk/highlight-next/package.json

```diff
@@ -7,14 +7,20 @@
 	"types": "./dist/index.d.ts",
 	"exports": {
 		".": {
+			"edge": "./dist/index.edge.js",
```

Comment:
Forking these runtimes to the `index.edge.js` noop.

See [vercel/otel](https://github.com/vercel/otel/blob/main/package.json) for the inspiration.